### PR TITLE
Adjust elasticsearch/elasticsearch version constraint in docs

### PR DESCRIPTION
**Pull request by nnatter**
When using `^7.9`, composer will install `7.10` when executing `composer update`

### CODE REVIEW

**Comment by alexander-schranz** [comment]
Can you rebase this to `2.1`?

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Execute the following [composer](https://getcomposer.org/) commands to add the b
 project:
 
 ```bash
-composer require elasticsearch/elasticsearch:^7.9 # should match version of your elasticsearch installation
+composer require elasticsearch/elasticsearch:7.9.* # should match version of your elasticsearch installation
```
Comment (alexander-schranz) [inline README.md:55]: Not all shells support * here so this is needed:

```suggestion
composer require elasticsearch/elasticsearch:"7.9.*" # should match version of your elasticsearch installation
```

File: Resources/doc/installation.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ The SuluArticleBundle requires a running elasticsearch `^5.0`, `^6.0` or `^7.0`.
 ## Install dependencies
 
 ```bash
-composer require elasticsearch/elasticsearch:^7.9 # should match version of your elasticsearch installation
+composer require elasticsearch/elasticsearch:7.9.* # should match version of your elasticsearch installation
```
Comment (alexander-schranz) [inline Resources/doc/installation.md:10]: Not all shells support * here so this is needed:

```suggestion
composer require elasticsearch/elasticsearch:"7.9.*" # should match version of your elasticsearch installation
```

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Comment by nnatter** [comment]
rebased 🙂

**Comment by alexander-schranz** [comment]
@nnatter Thank you!

**Pull request merged**